packages/util-body-length-browser/src/index.ts

```diff
@@ -1,6 +1,22 @@
 export function calculateBodyLength(body: any): number | undefined {
   if (typeof body === "string") {
-    return new Blob([body]).size;
+    let len = body.length;
+
+    // Source: https://github.com/DylanPiercey/byte-length
+    for (let i = len; i--; ) {
```

Comment:
I looked into the [Jsperf](https://jsperf.com/utf-8-byte-length/36) source code here's the code there:

```javascript
function byteLength(str) {
    // returns the byte length of an utf8 string
    var s = str.length;
    for (var i=str.length-1; i>=0; i--) {
      var code = str.charCodeAt(i);
      if (code > 0x7f && code <= 0x7ff) s++;
      else if (code > 0x7ff && code <= 0xffff) s+=2;
    }
    return s;
  }
```

To me this looks easier to read, especially for the `for` loop part.